src/device.jl

```diff
@@ -145,6 +145,13 @@ function device_type(agent::HSA.Agent)
     return devtype[]
 end
 
+wavefront_size(device::ROCDevice) = wavefront_size(device.agent)
+function wavefront_size(agent::HSA.Agent)
+    wfsize = Ref{UInt32}()
+    getinfo(agent, HSA.AGENT_INFO_WAVEFRONT_SIZE, wfsize) |> check
+    return wfsize[]
+end
+
```

Comment:
Can we call this `wavefrontsize` instead of `wavefront_size`? It should be possible to merge this with `Device.wavefrontsize()` in a future PR, so it'd be great if they had the same name.